services/node/node-lts.tester.js

```diff
@@ -0,0 +1,166 @@
+'use strict'
+
+const { expect } = require('chai')
+const { Range } = require('semver')
+const t = (module.exports = require('../tester').createServiceTester())
+const {
+  mockPackageData,
+  mockNonExistingPackageData,
+  mockReleaseSchedule,
+  mockVersionsSha,
+} = require('./testUtils/test-utils')
+
+function expectSemverRange(message) {
+  expect(() => new Range(message)).not.to.throw()
+}
+
```

Comment:
Same comment as in `node-current.tester.js` regarding live tests

Reply:
sure thing